pkg/api/types.go

```diff
@@ -1042,6 +1042,10 @@ type AzureFileVolumeSource struct {
 	// the ReadOnly setting in VolumeMounts.
 	// +optional
 	ReadOnly bool
+	// the namespace of the secret that contains Azure Storage Account Name and Key
+	// default is the same as the Pod
+	// +optional
+	SecretNameSpace string
```

Comment:
this is an interesting case, we already have secretname in the api. It is possible to have a backup object ref for secret but overriding secret name is odd.